genTriposGAFFandGAFF2.py

```diff
@@ -1,21 +1,22 @@
 #!/usr/bin/env python
 
+# Purpose: Generate Tripos/GAFF/GAFF2 mol2 files for purposes of
```

Comment:
@dkyu, let's add "# Written by: Victoria Lim and Daisy Kyu @ Mobley Lab UCI" at the top.